tests/unit/semantic_release/hvcs/test_github.py

```diff
@@ -559,9 +603,10 @@ def test_upload_asset_succeeds(
             == True
         )
         assert m.called
-        assert len(m.request_history) == 1
-        assert m.last_request.method == "POST"
-        assert m.last_request.url == "{url}?{params}".format(
+        assert len(m.request_history) == 2
+        post_req = m.last_request.copy()  # Reference of m.last_request is not fix
```

Comment:
What issues were you seeing with `m.last_request`? Aren't we examining the content of the last request regardless? Just curious why we need to take a copy